spec/formatting.md

```diff
@@ -222,6 +222,61 @@ the following steps are taken:
 
    The form that resolved _operand_ and _option_ values take is implementation-defined.
 
+   A _local-declaration_ binds the resolved value of an _expression_
+   to a _variable_.
+   Thus, the output of one _function_ is potentially the _operand_
+   of another _function_. In other words, formatting functions
+   compose with each other.
+   For example, in
+   ```
+   .input {$n :number minIntegerDigits=3}
+   .local {$n1 :number maxFractionDigits=3}
```

Comment:
As @aphillips noticed, the `.local` needs a name. Making this suggestion to make sure we don't miss it, in case the other review comment isn't committed.

```suggestion
   .input {$n :number minIntegerDigits=3}
   .local $x = {$n1 :number maxFractionDigits=3}
```